apps/server/src/trpc/index.ts

```
@@ -15,6 +15,8 @@ import { bimiRouter } from './routes/bimi';
 import type { HonoContext } from '../ctx';
 import { aiRouter } from './routes/ai';
 import { router } from './trpc';
+import { categoriesRouter } from './routes/categories';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'categoriesRouter'. Consider to delete it or rename it.

'categoriesRouter' is defined here:


(lint/suspicious/noRedeclare)